commcare_connect/opportunity/forms.py

```diff
@@ -103,35 +103,54 @@ class Meta:
         ]
 
     def __init__(self, *args, **kwargs):
-        kwargs["opportunity"] = kwargs.get(
-            "instance", None
-        )  # passing the opportunity instance to OpportunityUserInviteForm
         super().__init__(*args, **kwargs)
 
         self.helper = FormHelper(self)
         self.helper.layout = Layout(
-            Row(Field("name")),
-            Row(Field("active", css_class="form-check-input", wrapper_class="form-check form-switch")),
-            Row(Field("is_test", css_class="form-check-input", wrapper_class="form-check form-switch")),
-            Row(Field("delivery_type")),
-            Row(Field("description")),
-            Row(Field("short_description")),
             Row(
-                Field("currency", wrapper_class="form-group col-md-6 mb-0"),
-                Field("end_date", wrapper_class="form-group col-md-6 mb-0"),
+                HTML("<div class='col-span-2'><h6 class='title-sm'>Opportunity Details</h6>  <span class='hint'>Edit the details of the opportunity. All fields are mandatory.  </span> </div>"),
+                Column(
+                    Field("name", wrapper_class="w-full"),
+                    Field("short_description", wrapper_class="w-full"),
+                    Field("description", wrapper_class="w-full"),
+                ),
+                Column(
+                    Field("delivery_type"),
+                    Field("active", css_class=CHECKBOX_CLASS, wrapper_class="bg-slate-100 flex items-center justify-between p-4 rounded-lg"),
```

Comment:
is there a way to pull out these styles that get repeated so often, there are a few that show up multiple times and it would be good to have an easy way to keep them in sync